x-pack/legacy/plugins/siem/server/lib/detection_engine/notifications/rules_notification_alert_type.test.ts

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { savedObjectsClientMock } from 'src/core/server/mocks';
+import { loggerMock } from 'src/core/server/logging/logger.mock';
+import { getResult } from '../routes/__mocks__/request_responses';
+import { rulesNotificationAlertType } from './rules_notification_alert_type';
+import { buildSignalsSearchQuery } from './build_signals_query';
+jest.mock('./build_signals_query');
+
+describe('rules_notification_alert_type', () => {
+  const savedObjectsClient = savedObjectsClientMock.create();
```

Comment:
I would move these to a `beforeEach` if possible. The less shared state, the better.